metagpt/roles/qa_engineer.py

```diff
@@ -159,7 +166,7 @@ async def _act(self) -> Message:
         for msg in self._rc.news:
             # Decide what to do based on observed msg type, currently defined by human,
             # might potentially be moved to _think, that is, let the agent decides for itself
-            if msg.cause_by == WriteCode:
+            if msg.cause_by in [WriteCode, WriteCodeReview]:
```

Comment:
What was the reason for adding the WriteCodeReview? Perhaps it was added to fix some bugs?

Reply:
yes, fix bugs. Original code review won't trigger qa